src/telemetry.cpp

```diff
@@ -120,17 +120,17 @@ void Telemetry::initialize() {
           this, &Telemetry::onDaemonStatus);
 #endif
 
-  connect(controller, &Controller::handshakeFailed, this,
-          [](const QString& publicKey) {
-            logger.info() << "Send a handshake failure event";
-
-            mozilla::glean::sample::connectivity_handshake_timeout.record(
-                mozilla::glean::sample::ConnectivityHandshakeTimeoutExtra{
-                    ._server = publicKey,
-                    ._transport = MozillaVPN::instance()
-                                      ->networkWatcher()
-                                      ->getCurrentTransport()});
-          });
+  connect(
+      controller, &Controller::handshakeFailed, this,
+      [](const QString& publicKey) {
+        logger.info() << "Send a handshake failure event";
+        mozilla::glean::sample::connectivity_handshake_timeout.record(
+            mozilla::glean::sample::ConnectivityHandshakeTimeoutExtra{
+                ._server = publicKey,
+                ._transport = QVariant::fromValue(MozillaVPN::instance()
+                                                      ->networkWatcher()
+                                                      ->getReachability())});
```

Comment:
Please update the documentation for this metric. 

Note that usually this would not be an ok change -- changing values in telemetry, but since we are not looking at these metrics in any dashboard that I know of we can just go forward.